jdaviz/configs/default/plugins/model_fitting/tests/test_plugin.py

```diff
@@ -169,7 +169,7 @@ def test_fit_cube_no_wcs(cubeviz_helper):
     assert len(fitted_model) == 56  # ny * nx
     # Make sure shapes are all self-consistent within Cubeviz instance.
     fitted_data = cubeviz_helper.app.data_collection["model"]
-    assert fitted_data.shape == (8, 7, 9)  # nx, ny, nz
+    assert fitted_data.shape == (7, 8, 9)  # nx, ny, nz
```

Comment:
Either the comment here is wrong or the comment on L161 above is wrong. One of them has to change.
```suggestion
    assert fitted_data.shape == (7, 8, 9)  # ny, nx, nz
```

Reply:
I think 161 is wrong now.